lib/parsers.js

```diff
@@ -53,123 +53,105 @@ module.exports.parseError = function parseError(err, kwargs, cb) {
   });
 };
 
-module.exports.parseRequest = function parseRequest(req, kwargs) {
-  kwargs = kwargs || {};
+module.exports.parseRequest = function parseRequest(req, parseUser) {
```

Comment:
Not public; only callsite is currently in `errorHandler` middleware, soon we'll add another in `requestHandler` and in `process` but all internal. The `kwargs` param was unused so I got rid of it.